tests/core/pyspec/eth2spec/test/capella/block_processing/test_process_withdrawals.py

```diff
@@ -154,10 +163,14 @@ def test_success_max_per_slot(spec, state):
 
 
 @with_phases([CAPELLA])
+@with_presets([MAINNET], reason="too few validators with minimal config")
```

Comment:
Apologies, I was a bit confused when I last commented.

Suggest we do a case where `state.validators > MAX_VALIDATORS_PER_WITHDRAWALS_SWEEP` and the validators are all withdrawable

Both are valuable (as you had implicitly before between the two configs) but separating them out to be very clear which of the two scenarios is being tested is my preference

Reply:
if im following you correctly, then I've updated the PR to have each of these scenarios called out explicitly